### ISSUE TITLE
NTR: IGF2 low in blood (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8402]
New term request IGF2 low in blood (segal_170814153235): Somatomedin, insulin-like growth factor 2  UMLS: 4228600

**Comment by pnrobinson** [comment]
@MickeySegal
Adding new term: 
- ID: HP:6000404
- label: Reduced circulating insulin-like growth factor 2 concentration
- parents: [Abnormal circulating hormone concentration (HP:0003117)](https://hpo.jax.org/app/browse/term/HP:0003117)
- pmid: 
- def: The concentration of insulin-like growth factor 2 (IGF2) in the blood circulation is below the lower limit of normal. IGF2 is a peptide hormone regulating various cellular processes such as proliferation and apoptosis.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8402
- synonyms:
